Gfdl1.2 compliance (#78)
openSUSE/openSUSE-docs-revamped-temp: project/docs/zstd.md

## Costs & benefits
Compression is about making your files take less space on your hard drive, but it requires extra computations when reading or writing. In other words it trades (very marginally increased) processor and RAM usage for less hard drive usage. The tradeoff is likely to benefit the most modern machines with solid state drives (_SSD_), which are likely to have less storage capacity than their desktop counterparts, while having comparable computational power. 

> It should also be noticed that `zstd`compression can increase the lifespan of SSDs, as it may decrease reads over time.

Reviewer: While you're changing that line already ... Let's remove the redundant parts & add a space between "`zstd`compression".

```suggestion
`zstd` compression can increase the lifespan of SSDs, as it may decrease the number of reads.
```
